<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialog</class>
 <widget class="QDialog" name="TestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DVB Terrestrial Test</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>630</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>pressme</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>731</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Transponders/Antenna Adjust</string>
    </attribute>
    <widget class="QProgressBar" name="progressBarLevel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>561</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>561</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Transponders</string>
     </property>
     <widget class="QTableWidget" name="tableTranspondersList">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>321</width>
        <height>161</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Freq</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Symbol Rate</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modulation</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="buttonAddTransponder">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRemoveTransponder">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonStartLocking">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Locking</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStopLocking">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>230</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Locking</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLocked">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>330</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Signal Lock</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Signal Level</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Signal Quality</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBarQuality">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>380</y>
       <width>561</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Scan Spectrum</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>From </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFrom">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>300</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTo">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>854</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxStep">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>20</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz    to</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz    step</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAnalize">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxNonStop">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Non-Stop</string>
     </property>
    </widget>
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>671</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>GIS</string>
    </attribute>
    <widget class="QTableWidget" name="tablePoints">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>511</width>
       <height>101</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>alt</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>lat</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>lock</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>signal</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>20</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
    <widget class="QGCMapWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>661</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Constellation</string>
    </attribute>
    <widget class="QOpenGLWidget" name="openGLWidget">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>341</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QComboBox" name="comboBoxConstellRefStream">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxConstellFreq">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonObtainData">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Obtain Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference Stream</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxConstellNonStop">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>non-stop receiving</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>341</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualization Mode</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxConstallVizMod">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>QAM256</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QAM128</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QAM64</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBoxGroupNearest">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Group Nearest Points</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxRecovery">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Carrier Recovery</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DVB card:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCard">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>431</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>(none)</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QGCMapWidget</class>
   <extends>QWidget</extends>
   <header location="global">QGCMapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
